frontend/device/pocketbook/device.lua

```diff
@@ -203,6 +209,18 @@ local PocketBook740 = PocketBook:new{
     emu_events_dev = "/var/dev/shm/emu_events",
 }
 
+-- PocketBook Color Lux
+local PocketBookColorLux = PocketBook:new{
+    isTouchDevice = yes,
+    hasKeys = yes,
+    hasFrontlight = yes,
+    hasColorScreen = yes,
+    has3BytesWideFrameBuffer = yes,
+    isAlwaysPortrait = no,
+    display_dpi = 100,
```

Comment:
You can probably leave it out, fb:getDPI will fall back to 160, which should be roughly okay (it's was we do on other 600x800 devices) ;).